pages/AdListPage/[id]/index.js

```diff
@@ -51,22 +52,15 @@ export default function AdDetailsPage() {
         <p>Tags: </p>
         <TagsList>
           {ad.tags.map((tag, index) => {
-            return <TagItem key={index}> {tag} </TagItem>;
+            return <TagItem key={index + tag}> {tag} </TagItem>;
```

Comment:
You could also do it like this:

```suggestion
            return <TagItem key={tag}> {tag} </TagItem>;
```